src/Nerve_GIC/include/gudhi/GIC.h

```diff
@@ -71,7 +71,7 @@ using Weight_map = boost::property_map<Graph, boost::edge_weight_t>::type;
  *
  * \details
  * The data structure is a simplicial complex, representing a
- * Graph Induced simplicial Complex (GIC) or a Nerve,
```

Comment:
If I understand correctly, the idea is to remove the Nerve (cf. also `set_type` removal) ?
If this is the case, the [cover complex documentation](https://gudhi.inria.fr/doc/latest/group__cover__complex.html) needs to be reworked.
Also, do we keep src/Nerve_GIC/utilities/Nerve.cpp ?
Do we keep check_nerve in src/Nerve_GIC/test/test_GIC.cpp ?

Reply:
You are right, at the beginning I thought that Nerve is not really useful now that we have Mapper. But on a second thought, I realized that Nerve constructions can be actually more general (in particular since it can work for any cover, not only preimages of functions). So I put it back in my last commit ;-)